### DISCUSSION TITLE
Can Minio Enterprise support Object Gateway as a plugin or as such

### DISCUSSION THREAD

**Discussion by schwienbier** [minio/minio#17648] [Q&A] [answered]
Dear Minio Supports,

minio will deprecate the object gateway, according to this [blog](https://blog.min.io/deprecation-of-the-minio-gateway/). However, I would like to ask, if we purchase the license of enterprise Minio, would it be possible to use gateway feature as plugin, or as such? Thank you.

Best regards

**Comment by klauspost** [comment, marked as answer]
@schwienbier Please write `hello@min.io` and describe your case, and we can look at what options we have available.

**Comment by schwienbier** [comment]
hello@min.io

minio will deprecate the object gateway, according to this [blog](https://blog.min.io/deprecation-of-the-minio-gateway/). However, I would like to ask, if we purchase the license of enterprise Minio, would it be possible to use gateway feature as plugin, or as such? Thank you.

Best regards

**Reply by klauspost** [reply]
Please write an *email* to [hello@min.io](mailto:hello@min.io) and also include a description of what you are looking to do.
